src/utils/fee/index.ts

```diff
@@ -94,6 +94,87 @@ export const getStakingTxInputUTXOsAndFees = (
   };
 };
 
+/**
+ * Calculates the required funding UTXO and fees for a staking expansion transaction.
+ * 
+ * This function selects a single UTXO from available UTXOs to cover:
+ * 1. Transaction fees for the expansion
+ * 2. Any additional staking amount beyond the previous stake
+ * 
+ * @param availableUTXOs - List of available UTXOs to choose from for funding
+ * @param previousStakingTx - Details of the previous staking transaction being expanded
+ * @param stakingAmount - Total staking amount for the expansion (includes previous + additional)
+ * @param feeRate - Fee rate in satoshis per byte
+ * @param outputs - Transaction outputs for the expansion
+ * @returns Object containing the selected funding UTXO and calculated fee
+ */
+export const getStakingExpansionTxInputUTXOAndFees = (
```

Comment:
```suggestion
export const getStakingExpansionTxFundingUTXOAndFees = (
```
I think it's much more clear if we refer to the UTXO with its official name